src/EditorFeatures/Test/Workspaces/TestHostProject.cs

```diff
@@ -168,7 +169,7 @@ public string OutputFilePath
             IList<AnalyzerReference> analyzerReferences = null)
         {
             _assemblyName = assemblyName;
-            _name = assemblyName;
+            _name = projectName;
```

Comment:
I agreed at first, and but this field is exposed through the Name property, and saying Project.Name makes sense. It's the name of the project, after all. :-)